### ISSUE TITLE
[Bug]: Unable to add this feed - certificate issue

### ISSUE THREAD

**Issue by pavelvlcek** [nvdaes/readFeeds#61]
### Summary

I am unable to add this specific feed. A lot of rss readers have issues with this specific feed, including Nvda addon.

### To reproduce

Try to add this feed: https://tn.nova.cz/feed/atom/tnnova-2
or
https://tn.nova.cz/rss
and you will get error with ssl certificate ssl.C1006, it is not possible to copy full error message to clipboard.
Sometimes, it is possible to add his feed without any issue, but I don't know, why sometimes works as expected and sometimes not.

### Expected behavior

Add this feed to list of feeds as expected.

### Relevant log output

```shell

```

### Version

43.1.0

**Comment by nvdaes** [comment]
Hello:
I cannot reproduce this issue on my System, with NVDA 2025.3rc1.

Anyway, the provided feeds are not valid according to

https://validator.w3.org/feed

I suggest you to contact with the authors of these feeds to fix this problem.
Let's close this issue for now. If these feeds are fixed, and you have issues, please reopen this or create a new issue.
Thanks for your interest in the readFeeds add-on.
